edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Test/packages.lock.json

```diff
@@ -496,8 +483,8 @@
       },
       "Microsoft.Rest.ClientRuntime": {
         "type": "Transitive",
-        "resolved": "2.3.21",
-        "contentHash": "KDYlgTyO693V6pi6SGk9eg+dDvKjuOgmkapbHdpnB1SmTPKpvWxVLIMyARJsCFLfB6axyURUJHOfvxBQ0yJKeg==",
+        "resolved": "2.3.20",
+        "contentHash": "bw/H1nO4JdnhTagPHWIFQwtlQ6rb2jqw5RTrqPsPqzrjhJxc7P6MyNGdf4pgHQdzdpBSNOfZTEQifoUkxmzYXQ==",
```

Comment:
Is the downgrade intentional here?

Reply:
No, that must have happened automatically when the Microsoft.Azure.Devices package was removed from Edge Agent. With that package gone, the only package that still requires Microsoft.Rest.ClientRuntime in this project is Microsoft.Azure.KeyVault, which specifies a range rather than a specific version. So .NET resolved an older version of the dependency.